### PR TITLE
Show footer contracts on mobile.

### PR DESCRIPTION
**Pull request by biocom**
Addressing #310

- Shows the contract links on mobile.
- Changed text color from orange to dark green for better readability/contrast.

Notes:
- Noticed a 'phantom' element/button exists, which creates an artificial white space area. We need to address this in a separate PR. Seems to be component `</ButtonEmpty>` in `src/custom/components/swap/UnsupportedCurrencyFooter/UnsupportedCurrencyFooterMod.tsx`

Screen Shot 2021-04-27 at 14 50 34

Screen Shot 2021-04-27 at 14 50 43

Screen Shot 2021-04-27 at 14 50 50

Phantom element (likely `</ButtonEmpty>`)
Screen Shot 2021-04-27 at 14 44 17
Screen Shot 2021-04-27 at 14 42 41

### CODE REVIEW

File: src/custom/theme/cowSwapTheme.tsx [hunk 1]
```diff
@@ -56,8 +56,6 @@ export function colors(darkMode: boolean): Colors {
     secondary3: darkMode ? '#17000b26' : 'rgba(137,88,255,0.6)',
 
     // ****** other ******
-    blue1: '#3F77FF',
```
Comment (biocom) [inline src/custom/theme/cowSwapTheme.tsx:59]: Removed, because inherited from the baseTheme.

File: src/custom/components/Version/index.tsx [hunk 2]
```diff
@@ -61,21 +61,18 @@ const StyledPolling = styled.div`
   display: flex;
   flex-flow: column nowrap;
   align-items: flex-start;
-
-  padding: 1rem;
```
Comment (biocom) [inline src/custom/components/Version/index.tsx:65]: Consistent use of px's instead of rem.

File: src/custom/components/Footer/index.tsx [hunk 3]
```diff
@@ -15,13 +22,14 @@ const Wrapper = styled.div`
   justify-content: space-evenly;
   margin: auto 96px 0 32px;
   width: 100%;
+
+  ${({ theme }) => theme.mediaWidth.upToSmall`
+    margin: 0 auto 150px;
```
Comment (biocom) [inline src/custom/components/Footer/index.tsx:27]: Adds margin below the contract links footer element, so the user can scroll down and see this area.

**Review by anxolin — APPROVED** [review]

**Review by alfetopito — COMMENTED** [review]
I don't see it unless I make the width narrower.

Narrow | Narrower
:-:|:-:
screenshot_2021-04-27_07-23-41 | screenshot_2021-04-27_07-23-50

**Comment by anxolin** [comment]
@alfetopito u are right. Is broken for resolution between 990 and 720 aprox!

Good catch, I tried desktop and mobile only

**Review by alfetopito — APPROVED** [review]
Awesome! Now I can see it all the time, approved :)

Still one small note, that is not a blocker.

The opacity is different depending on the screen size.
Intentional?

Small | Smaller
:-:|:-:
screenshot_2021-04-27_09-19-04 | screenshot_2021-04-27_09-19-15

**Comment by biocom** [comment]
> Awesome! Now I can see it all the time, approved :)
> 
> Still one small note, that is not a blocker.
> 
> The opacity is different depending on the screen size.
> Intentional?
> 

Thanks for reviewing. Correct, this is intentional due to the tap (mobile) vs. hover (desktop) experience.

**Review by anxolin — APPROVED** [review]

**Pull request merged**